src/dms/tests/EdFi.DataManagementService.Tests.E2E/Features/General/ReadValidation.feature

```diff
@@ -0,0 +1,110 @@
+                  | schoolId  | nameOfInstitution            | gradeLevels                                                                         | educationOrganizationCategories                                                                                   |
+                  | 255901044 | Grand Bend Middle School     | [ {"gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#Sixth grade"} ]    | [ {"educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor#School"} ] |
+                  | 255901107 | Grand Bend Elementary School | [ {"gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#First grade"} ]    | [ {"educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor#School"} ] |
+                  | 255901634 | Grand Bend High School       | [ {"gradeLevelDescriptor": "uri://ed-fi.org/GradeLevelDescriptor#Eleventh grade"} ] | [ {"educationOrganizationCategoryDescriptor": "uri://ed-fi.org/EducationOrganizationCategoryDescriptor#School"} ] |
+            Given the system has these "students"
+                  | studentUniqueId | birthDate  | firstName | lastSurname |
+                  | 604829          | 2010-01-13 | Traci     | Mathews     |
+                  | 604829          | 2015-08-17 | April     | Shelton     |
+
+        @ignore
+        Scenario: 01 Ensure that a resource can be retrieved by name
+             When a GET request is made to "/ed-fi/students?firstName=April"
+             Then it should respond with 200
+              And the response body is
+                  """
+                  [
+                      {
+                          "studentUniqueId": "604829",
+                          "birthDate": "2015-08-17",
+                          "firstName": "April",
+                          "lastSurname": "Shelton",
+                          "_lastModifiedDate": "2024-05-10T20:57:00.9107017Z"
+                      }
+                  ]
+                  """
+
+        @ignore
+        Scenario: 02 Ensure that a resource can be retrieved bt lastModifiedDate property
+             When a GET request is made to "/ed-fi/students?_lastModifiedDate=2024-07-09T18:46:39.5385642Z"
+             Then it should respond with 200
+              And the response body is
+                  """
+                  [
+                      {
+                          "studentUniqueId": "604824",
+                          "birthDate": "2010-01-13",
+                          "firstName": "Traci",
+                          "lastSurname": "Mathews",
+                          "_lastModifiedDate": "2024-07-09T18:46:39.5385642Z"
+                      }
+                  ]
+                  """
+
+        @ignore
+        Scenario: 03 Ensure that resources can be ordered by attribute ascending or descending
+             When a POST request is made to "/ed-fi/schools?orderBy=schoolId&direction=desc"
```

Comment:
You are missing the with at the end

Reply:
This was a mistake, this is a GET request, changed